torch_xla/csrc/lowering_context.h

```diff
@@ -71,6 +76,11 @@ class LoweringContext : public torch::lazy::LoweringContext {
   // embedded XLA builder (returned by the builder() API).
   xla::StatusOr<xla::XlaComputation> BuildXla();
 
+  // Build the XLA computation capturing all the operations created with the
+  // embedded XLA builder (returned by the builder() API) with check whether
+  // build for cond/body computation or not.
+  xla::StatusOr<xla::XlaComputation> BuildXlaWithCheck();
```

Comment:
Can you merge the logic to the original `BuildXla()`, since you can condition on the designated computation names?

Reply:
Thanks, make sense, we used to write a duplicated BuildXla() to avoid affect original `BuildXla()` performance, and tested locally&simply: modify `BuildXla` has no appear affect on performance, merged the logic to the original BuildXla()